roles/template-configs/vars/main.yml

```diff
@@ -0,0 +1,6 @@
+browbeat_elastic_enabled: false
+browbeat_elastic_host: "1.2.3.4.5"
+browbeat_grafana_enabled: false
+browbeat_grafana_host: "1.2.3.4.5"
```

Comment:
So the CI Job will replace these vars per job?

Reply:
Yes exactly while calling browbeat it'll replace the vars, something like http://git.app.eng.bos.redhat.com/git/rhos-qe-jenkins.git/tree/jobs/defaults/stages/ir_browbeat.groovy.inc